src/commons/services/block-period-service.js

```
@@ -0,0 +1,147 @@
+      bookableId,
+      timeBegin,
+      timeEnd,
+    );
+    const provider = new ContextDataProvider(context);
+
+    const blockPeriods = [];
+
+    for (const instance of instances) {
+      const availability = await checkWindowAvailabilityModule.checkWindowAvailability(
+        provider,
+        {
```

ESLint

[error] Replace `·await·checkWindowAvailabilityModule.checkWindowAvailability(⏎········provider,⏎·······` with `⏎········await·checkWindowAvailabilityModule.checkWindowAvailability(provider,`

(prettier/prettier)